ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/DC1-BL1A.yml

```diff
@@ -261,7 +261,14 @@ management_api_http:
   enable_https: false
   default_services: false
   enable_vrfs:
-  - name: MGMT
+  - name: VRF_Enable
```

Comment:
could we create new hosts for management-eapi-settings for this?

Reply:
Done!